README.md

```diff
@@ -43,55 +43,54 @@ The following examples assume you are using a kotlin build script ([which are mu
     }
     ```
 
-### Features
-
-- You can automatically add Mindustry dependencies with :
+3. Automatically add Mindustry dependencies with :
 
-  ```kotlin
-  import fr.xpdustry.toxopid.util.anukenJitpack
-  import fr.xpdustry.toxopid.util.mindustryDependencies
-
-  repositories {
-      mavenCentral()
-      anukenJitpack()
-  }
+    ```kotlin
+    import fr.xpdustry.toxopid.util.anukenJitpack
+    import fr.xpdustry.toxopid.util.mindustryDependencies
 
-  dependencies {
-      mindustryDependencies()
-  }
-  ```
+    repositories {
+        mavenCentral()
+        anukenJitpack()
+    }
 
-- You can use the info of your `[mod|plugin].[h]json` in your build script with `ModMetadata` :
+    dependencies {
+        mindustryDependencies()
+    }
+    ```
 
-  ```kotlin
-  import fr.xpdustry.toxopid.util.ModMetadata
-  
-  val metadata = ModMetadata.fromJson(project.file("mod.hjson"))
-  // Setting the project version from the one located in "mod.json"
-  project.version = metadata.version
-  ```
+4.  Load the info of your `[mod|plugin].[h]json` in your build script with `ModMetadata` :
+    ```kotlin
+    import fr.xpdustry.toxopid.util.ModMetadata
 
-  or directly generating your `[mod|plugin].[h]json` inside the build script and include it in the jar :
+    val metadata = ModMetadata.fromJson(project.file("mod.hjson"))
+    // Setting the project version from the one located in "mod.json"
+    project.version = metadata.version
+    ```
+    or directly generate your `[mod|plugin].[h]json` from your build script :
 
-  ```kotlin
-  import fr.xpdustry.toxopid.util.ModMetadata
-  
-  val metadata = ModMetadata(
-      name = "example",
-      displayName = "Example",
-      description = "A very nice mod :)",
-      main = "com.example.mod.ModMain"
-  )
-
-  tasks.jar {
-      // Doing it in doFirst makes sure it's only executed when this task runs
-      doFirst {
-          val temp = temporaryDir.resolve("mod.json")
-          temp.writeText(metadata.toJson(true))
-          from(temp)
-     }
-  }
-  ```
+    ```kotlin
+    import fr.xpdustry.toxopid.util.ModMetadata
+
+    val metadata = ModMetadata(
+        name = "example",
+        displayName = "Example",
+        description = "A very nice mod :)",
+        main = "com.example.mod.ModMain"
+    )
+    ```
+5. Include your `[mod|plugin].[h]json` into the build jar
+    ```kotlin
+    tasks.jar {
+        // Doing it in doFirst makes sure it's only executed when this task runs
+        doFirst {
+            val temp = temporaryDir.resolve("mod.json")
+            temp.writeText(metadata.toJson(true))
+            from(temp)
+        }
+    }
+    ```
```

Comment:
```suggestion
5. Include the generated `ModMetadata` in your mod/plugin into the build jar
    ```kotlin
    tasks.jar {
        // Doing it in doFirst makes sure it's only executed when this task runs
        doFirst {
            val temp = temporaryDir.resolve("mod.json")
            temp.writeText(metadata.toJson(true))
            from(temp)
        }
    }
    ```
    
    > If you did not generated or modified the mod metadata in your build script, simply include the local `[mod|plugin].[h]json` with `from(file("mod.json"))`.
```
Including the `ModMetadata` is not mandatory.